src/librustc_errors/lib.rs

```diff
@@ -501,9 +501,8 @@ impl Handler {
 
                 return;
             }
-            1 => s = "aborting due to previous error".to_string(),
             _ => {
-                s = format!("aborting due to {} previous errors", self.err_count.get());
+                s = format!("aborting due to previous error(s)".to_string);
```

Comment:
The call to `to_string` should just be removed (or made into a call and the `format!` removed.